Packs/ThreatConnect/Integrations/ThreatConnect_v2/README.md

```diff
@@ -0,0 +1,2406 @@
+		"Vendor": "ThreatConnect",
+		"Indicator": "88.88.88.88",
+		"Score": 1,
+		"Type": "ip" } ]
+}
+```
+
+#### Human Readable Output
+
+>### ThreatConnect IP Reputation for: 88.88.88.88
+>|Confidence|Create Date|ID|Last Modified|Name|Owner|Rating|Type|
+>|---|---|---|---|---|---|---|---|
+>| 0 | 2020-04-27T04:57:20Z | 112677927 | 2020-04-27T04:57:20Z | 88.88.88.88 | Demisto Inc. | 0 | Address |
+
+
+### url
+***
+Searches for an indicator of type URL.
+
+
+#### Base Command
+
+`url`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| url | The URL for which to search. For example, "www.demisto.com". | Required | 
+| owners | A comma-separated list of a client's organizations, sources, or communities to which a client’s API user has been granted permission. For example, "owner1", "owner2", or "owner3". | Optional | 
+| ratingThreshold | A list of results filtered by indicators whose threat rating is greater than the specified value. Can be "0" - "Unknown", "1" - "Suspicious", "2" - "Low", "3" - Moderate, "4" - High, or "5" - "Critical". | Optional | 
+| confidenceThreshold | A list of results filtered by indicators whose confidence rating is greater than the specified value. Can be "0%" - "Unknown," "1% " - "Discredited", "2-29%" - "Improbable," "30-49%" - "Doubtful," "50-69%" - "Possible", "70-89%" - "Probable," or "90-100%" - "Confirmed". | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| TC.Indicator.Name | string | The name of the indicator. | 
+| TC.Indicator.Type | string | The type of the indicator. | 
+| TC.Indicator.ID | string | The ID of the indicator. | 
+| TC.Indicator.Description | string | The description of the indicator. | 
+| TC.Indicator.Owner | string | The owner of the indicator. | 
+| TC.Indicator.CreateDate | date | The date on which the indicator was created. | 
+| TC.Indicator.LastModified | date | The date on which the indicator was last modified. | 
+| TC.Indicator.Rating | number | The threat rating of the indicator. | 
+| TC.Indicator.Confidence | number | The confidence rating of the indicator. | 
+| DBotScore.Indicator | string | The value assigned by DBot for the indicator. | 
+| DBotScore.Type | string | The type assigned by DBot for the indicator. | 
+| DBotScore.Score | number | The score assigned by DBot for the indicator. | 
+| DBotScore.Vendor | string | The vendor used to calculate the score. | 
+| URL.Data | string | The data of the URL indicator. | 
+| URL.Malicious.Vendor | string | For malicious URLs, the vendor that made the decision. | 
+| URL.Malicious.Description | string | For malicious URLs, the full description. | 
+
+
+#### Command Example
+```!url url=https://www.domain.com```
+
+#### Context Example
+```
+{}
+```
+
+#### Human Readable Output
+
+>Make sure that the indicator exists in your ThreatConnect environment### ThreatConnect URL Reputation for: https://www.domain.com
+>**No entries.**
```

Comment:
this doesn't look good, make sure it looks good in the war-room.
switch to a URL with details.